examples/mysql-galera/image/docker-entrypoint.sh

```diff
@@ -0,0 +1,186 @@
+#!/bin/bash
+
+# Copyright 2015 The Kubernetes Authors All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# 
+# This script does the following:
+# 
+# 1. Sets up database privileges by building an SQL script
+# 2. MySQL is initially started with this script a first time 
+# 3. Modify my.cnf and cluster.cnf to reflect available nodes to join
+# 
+
+HOSTNAME=`hostname`
+# if NUM_NODES not passed, default to 3
+if [ -z "$NUM_NODES" ]
+then
```

Comment:
hehe - I changed it to that thinking it was the desired style. I'll add that bash style guide as a bookmark.